internal/configs/sharding/mainnet.go

```diff
@@ -10,6 +10,8 @@ const (
 	mainnetEpochBlock1 = 327680 // 20 * 2^14
 	blocksPerShard     = 16384  // 2^14
 	mainnetV1Epoch     = 1
+
+	mainnetVdfDifficulty = 100000 // This takes about 200s to finish the vdf
```

Comment:
In the future our leader rotate every 40 blocks which is around 200s with 5 second block time. So better to make the vdf difficulty around 100s or 2 minutes